<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>putong</class>
 <widget class="QDialog" name="putong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>普通模式—考试指令播放系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ico.qrc">
    <normaloff>:/C:/Users/Administrator/Desktop/logo-dc/logo128.ico</normaloff>:/C:/Users/Administrator/Desktop/logo-dc/logo128.ico</iconset>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>3</number>
   </property>
   <property name="text">
    <string>DC作品</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #aaa;
border-radius:15px;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>12</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>考试指令播放系统（普通模式）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>请选择考试科目：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>语文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>思想政治</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>外语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物理/历史</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>信息/通用技术</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>化学/地理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>生物</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>历史</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>信息技术</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通用技术</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>90</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>考试开始时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>考试结束时间：</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="startTime">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="time">
     <time>
      <hour>8</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QTimeEdit" name="endTime">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>120</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="time">
     <time>
      <hour>10</hour>
      <minute>30</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ico.qrc"/>
 </resources>
 <connections/>
</ui>
